### PR TITLE
A tutorial on how to create a 3Box chatbox plugin

### PR DESCRIPTION
**Pull request by writeprovidence**
An Outline on the article, please take a look.

1). An overview of 3box chatbox plugin
2). Introduction to 3Box chatbox plugin
3). Use cases of 3Box chatbox plugin
4). How 3box chatbox works
(i) example of 3box chatbox plugin
5). How to build a 3box chatbox plugin.

### CODE REVIEW

**Comment by writeprovidence** [comment]
@ChrisChinchilla  an outline on how to create a 3Box chatbox plugin

**Comment by ChrisChinchilla** [comment]
Still would have liked a bit more detail @writeprovidence but OK…

File: A tutorial on how to create a 3Box chatbox plugin.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Content
+A tutorial on how to create a 3Box chatbox plugin
+
+Outline 
+
+1). An overview of 3box chatbox plugin
+2). Introduction to 3Box chatbox plugin
```
Comment (ChrisChinchilla) [inline A tutorial on how to create a 3Box chatbox plugin.md]: @writeprovidence What's the difference between points 1, 2 and 3?

Maybe start with a general, but brief introduction to 3Box plugins first as point 1? And then point 2 is specific to the chatbox plugin?

File: A tutorial on how to create a 3Box chatbox plugin.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+3). Use cases of 3Box chatbox plugin
+4). How 3box chatbox works
+(i) example of 3box chatbox plugin
+5). How to build a 3box chatbox plugin.
```
Comment (ChrisChinchilla) [inline A tutorial on how to create a 3Box chatbox plugin.md]: @writeprovidence What's the difference between points 4) i. and 5?

Comment (ChrisChinchilla) [inline A tutorial on how to create a 3Box chatbox plugin.md]: @writeprovidence What will you build?

File: A tutorial on how to create a 3Box chatbox plugin.md (hunk 1 above)
Comment (writeprovidence) [reply A tutorial on how to create a 3Box chatbox plugin.md]: Yes @ChrisChinchilla , that is the reason why i made it that way. i don't know if its fine by you.

File: A tutorial on how to create a 3Box chatbox plugin.md (hunk 2 above)
Comment (writeprovidence) [reply A tutorial on how to create a 3Box chatbox plugin.md]: A feedback system for dapps based on ethereum network.

Comment (writeprovidence) [reply A tutorial on how to create a 3Box chatbox plugin.md]: 4 i) Will be making reference to an already existing chatbox plugin, while (5) will be detaining the process of building a 3box chatbox plugin.

Comment (writeprovidence) [reply A tutorial on how to create a 3Box chatbox plugin.md]: @ChrisChinchilla still waiting for your feedback

Comment (writeprovidence) [reply A tutorial on how to create a 3Box chatbox plugin.md]: @ChrisChinchilla , i noticed a change of title to  (A tutorial on how to create an application using 3Box chatbox plugin)

File: A tutorial on how to create a 3Box chatbox plugin.md (hunk 1 above)
Comment (ChrisChinchilla) [reply A tutorial on how to create a 3Box chatbox plugin.md]: OK, that doesn't really answer my question, could you update the outline to clarify please?

File: A tutorial on how to create a 3Box chatbox plugin.md (hunk 2 above)
Comment (ChrisChinchilla) [reply A tutorial on how to create a 3Box chatbox plugin.md]: OK, sounds good.

Comment (ChrisChinchilla) [reply A tutorial on how to create a 3Box chatbox plugin.md]: Yup, that was at the request of the 3box team, they wanted to make it clearer. Could you add those details to the outline please?

**Review by ChrisChinchilla — CHANGES_REQUESTED** [review]
Just need a bit more detail to ensure that the content is what we're looking for and we're good to go

**Comment by writeprovidence** [comment]
okay @ChrisChinchilla will do that.

**Comment by ChrisChinchilla** [comment]
Any progress @writeprovidence ?

**Comment by writeprovidence** [comment]
@ChrisChinchilla yes, still working on it, will send in an outline that reflex the current change

**Comment by writeprovidence** [comment]
@ChrisChinchilla I made some iteration on the outline, please take a look. working on this direction already.

1. Introduction to 3box chatbox plugin.
2. use cases of 3box chatbox plugin.
3. How 3box chatbox works.
 (i) example of 3box chatbox plugin
4. How to build with 3box chatbox plugins
5. Reasons for using 3box chatbox plugins

**Comment by ChrisChinchilla** [comment]
@writeprovidence I don't see any updates in the PR, you need to make commits to make updates.

**Comment by writeprovidence** [comment]
ok

**Pull request closed without merging**